app/src/main/java/com/sunlightlabs/android/congress/notifications/subscribers/ActionsBillSubscriber.java

```diff
@@ -16,7 +16,7 @@
 
 	@Override
 	public String decodeId(Object result) {
-		return String.valueOf(((Bill.Action) result).acted_at.getTime());
+		return String.valueOf(((Bill.Action) result).acted_on.getTime());
```

Comment:
Oh, yikes - noticing this, I'll have to find a better way of distinguishing actions from each other with unique IDs, since we no longer have specific times for actions. One idea is maybe action order? Indexed from the oldest first, to stay consistent? That doesn't come in the API, so I'd have to store it on the `Bill.Action` during parse-time.